From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Thu, 28 May 2020 09:41:04 -0400
Subject: disable unused pip

Forwarded: not-needed

Remove pip from setup.cfg because it will not be used to build this
Debian package
---
 setup.cfg | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index ebe9506..4852d73 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -37,7 +37,6 @@ project_urls =
 include_package_data = true
 python_requires = >=3.7
 setup_requires =
-  pip >= 10
 install_requires =
   appdirs
   paramiko
